src/applications/mhv-medical-records/components/shared/NewRecordsIndicator.jsx

```diff
@@ -162,9 +162,36 @@ const NewRecordsIndicator = ({
   };
 
   return (
-    <div className="vads-u-margin-y--2  no-print" id="new-records-indicator">
-      {content()}
-    </div>
+    <>
+      <div className="vads-u-margin-y--2 no-print" id="new-records-indicator">
+        {content()}
+      </div>
+      <div className="print-only">
+        {!lastSuccessfulUpdate && (
+          <span>
```

Comment:
Rather than use `<span>` to encapsulate everything, you can just use a React fragment (`<>`)